app/src/htmlBackend/java/com/stxnext/intranet2/backend/api/EmployeesApiImpl.java

```diff
@@ -169,6 +170,11 @@ public void onAbsenceDaysLeftReceived(int mandated, int days, int absenceDaysLef
                     }
 
                     @Override
+                    public void onTimeReportReceived(List<TimeReportDay> timeReportDays, Calendar month) {
+
+                    }
```

Comment:
This could be all one-line empty implementations. It's taking too much space in the class right now and does nothing.